hypershift-operator/controllers/nodepool/inplace.go

```diff
@@ -171,6 +171,7 @@ func (r *NodePoolReconciler) reconcileMachineSet(ctx context.Context,
 	var status corev1.ConditionStatus
 	reason := ""
 	message := ""
+	status = "checking..."
```

Comment:
I'm not sure what this is coming from. Condition Status can either be true/false/unknown.

Reply:
maybe unknown it's better pick. Thanks